fuel/app/classes/presenter/welcome/404.php

```diff
@@ -16,6 +16,27 @@ class Presenter_Welcome_404 extends Presenter
 	 */
 	public function view()
 	{
+		$fufufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fddufufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fufddufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fufufddddu = "jjkpowek;rlwke;lk;lk;lk";
+
+
+		$fufu123fddddu = "jjkpowek;rlwke;lk;lk;lk";
+
+		$fufufdd333ddu = "jjkpowek;rlwke;lk;lk;lk";
+		$fufufddddu33 = "jjkpowek;rlwke;lk;lk;lk";
+
+		$fufufdddd111u = "jjkpowek;rlwke;lk;lk;lk";
+
+		$fufufddd123123d111u = "jjkpowek;rlwke;lk;lk;lk";
+		$fufufdddasd111u = "jjkpowek;rlwke;lk;lk;lk";
+
+		$fufuddddfdddd111u = "jjkpowek;rlwke;lk;lk;lk";
+
+		$fdddufufdddd111u = "jjkpowek;rlwke;lk;lk;lk";
```

Comment:
__[code_sniffer]__
* String "jjkpowek;rlwke;lk;lk;lk" does not require double quotes; use single quotes instead